### ISSUE TITLE
Feedback on Erythrite elites

### ISSUE THREAD

**Issue by SOMSTATE** [TheMysticSword/EliteVariety#45]
After playing with this mod for a few months with 4 friends, personally, we feel Erythrites seem far too strong in their current state. Hitscan chip damage from wisps and other low damage enemies become extremely high damage and impale you for extended amounts of damage, considering these are elites that can and have spawned on stage 3, maybe before but I haven't witnessed it.

I'd suggest the impale either repeating+increasing the damage of the hit that caused it over time, or making them spawn a bit later in the run, ideally stage 5. They are a valuable addition and I enjoy the design and mechanics, however, on stage 3 where you have maybe 3-4 conditional healing items if you're lucky, they can easily out damage them and kill you from the 75% hp (give or take) the initial activation of impaling leaves you on.

**Comment by SalTheThief** [comment]
Agreed. Me and my friend have had multiple early God runs end abruptly due to the impaling damage. We weren't even sure of what was doing the damage, so maybe make the feedback for that a little more obvious as well?
